### PR TITLE
Update/footer tweaks

### PR DESCRIPTION
**Pull request by niedzielski**

### CODE REVIEW

**Comment by reetssydney** [comment]
hi @niedzielski – Android currently footer links in the center so I updated new footer design with logo centered accordingly. We can update to left-aligned though for consistency with iOS and mobile web.

**Comment by montehurd** [comment]
@reetssydney oops I misspoke - I double checked with `a` designer. My bad!

**Comment by niedzielski** [comment]
This is a WIP with other changes present and some coming too. I will amend the design change.

File: src/transform/FooterReadMore.js [hunk 1]
```diff
@@ -169,29 +169,29 @@ const showReadMorePages = (pages, containerID, saveButtonClickHandler, titlesSho
...
-  wbptterms: 'description',
-  formatversion: 2
+  gsrlimit: count, // Limit search results by count.
+  gsrprop: 'redirecttitle', // Include a a parsed snippet of the redirect title property.
+  gsrsearch: `morelike:${title}`, // Weight search with the title.
+  gsrwhat: 'text', // Search the text then titles of pages.
+
+  // https://www.mediawiki.org/wiki/Extension:TextExtracts
+  exchars: 256, // Limit number of characters returned.
+  exintro: '', // Only content before the first section.
```
Comment (montehurd) [inline src/transform/FooterReadMore.js]: Should `exintro` be 1?

File: src/transform/FooterReadMore.js [hunk 2]
```diff
@@ -169,29 +169,29 @@ const showReadMorePages = (pages, containerID, saveButtonClickHandler, titlesSho
...
-  gsrsearch: `morelike:${title}`,
-  gsrwhat: 'text',
-  ns: 'ppprop',
-  pilimit: count,
-  piprop: 'thumbnail',
-  pithumbsize: 120,
-  prop: 'pageterms|pageimages|pageprops|revisions|extracts',
-  rrvlimit: 1,
-  rvprop: 'ids',
-  wbptterms: 'description',
```
Comment (montehurd) [inline src/transform/FooterReadMore.js]: Was `wbptterms` left out intentionally?

Comment (montehurd) [inline src/transform/FooterReadMore.js]: iirc we need it to get the wikidata description...

Comment (berndsi) [inline src/transform/FooterReadMore.js]: `wbptterms` is used to restrict `pageterms` to just the description. If omitted you'd get all of alias, description, label. See https://en.wikipedia.org/w/api.php?action=help&modules=query%2Bpageterms

If you only need the description then it's probably better to use `wbptterms=description`.

File: src/transform/FooterReadMore.css [hunk 3]
```diff
@@ -89,11 +89,12 @@ a.pagelib_footer_readmore_page:hover {
   color: #69f;
 }
 
-html[dir='rtl'] .pagelib_footer_readmore_page_save {
+html[dir='rtl'] .pagelib_footer_readmore_page_save,
+.content-rtl .pagelib_footer_readmore_page_save {
```
Comment (montehurd) [inline src/transform/FooterReadMore.css]: Oh, why is `.content-rtl` replacing `html[dir=rtl]`?

**Comment by montehurd** [comment]
Closing for now - added note to circle back and re-review later.

**Pull request closed without merging**